src/test/java/com/myFirstProject/myFirstProject/service/CurrencyRateSchedulerTest.java

```diff
@@ -0,0 +1,47 @@
+package com.myFirstProject.myFirstProject.service;
+
+import com.myFirstProject.myFirstProject.model.CurrencyRate;
+import com.myFirstProject.myFirstProject.repository.CurrencyRateRepository;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.*;
+import static org.mockito.ArgumentMatchers.eq;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CurrencyRateSchedulerTest {
+    @Mock
+    private CurrencyRateRepository currencyRateRepository;
+
+    @Mock
+    private CurrencyRateService currencyRateService;
+
+    @Test
+    public void updateCurrencyRate() {
+//        GIVEN
+        CurrencyRateScheduler currencyRateScheduler = new CurrencyRateScheduler();
+        currencyRateScheduler.setCurrencyRateRepository(currencyRateRepository);
+        currencyRateScheduler.setCurrencyRateService(currencyRateService);
+
+        Mockito.when(currencyRateService.getRate()).thenReturn(getListCurrencyRate());
+//        WHEN
+        currencyRateScheduler.updateCurrencyRate();
+//        THEN
+        Mockito.verify(currencyRateRepository, Mockito.times(1)).saveAll(eq(getListCurrencyRate()));
```

Comment:
Mockito.times(1) - can be skiped as it is default implementation for single argument